src/db/questionOptions.ts

```diff
@@ -20,6 +22,10 @@ export const questionOptions = pgTable('question_options', {
 });
 
 export const questionOptionsRelations = relations(questionOptions, ({ one, many }) => ({
+  user: one(users, {
+    fields: [questionOptions.userId],
+    references: [users.id],
+  }),
```

Comment:
add relation on user as well

Reply:
added relation to `users` table as well.